src/python/espressomd/electrostatics.pxd

```diff
@@ -153,27 +135,26 @@ IF ELECTROSTATICS:
 
             return p3m_set_params(r_cut, mesh, cao, alpha, accuracy)
 
-        cdef inline python_p3m_set_tune_params(p_r_cut, p_mesh, p_cao, p_alpha, p_accuracy, p_n_interpol):
+        cdef inline python_p3m_set_tune_params(p_r_cut, p_mesh, p_cao, p_alpha, p_accuracy):
             # cdef inline python_p3m_set_tune_params():
```

Comment:
here and in
https://github.com/espressomd/espresso/blob/7f90057b87cb5b676a29fb42dbb7897157c04b95/src/python/espressomd/magnetostatics.pyx#L233
do these comments serve any purpose?

Reply:
No idea, probably not.